src/components/NavigationBar/style.css

```diff
@@ -11,12 +11,15 @@
 }
 
 .fullScreenNav {
-  position: absolute;
-  .back {
-    color: white;
-  }
-  .iconBack {
-    background-image: url('assets/chevron-right-white.svg');
+  @media (--small-viewport) {
+    background: linear-gradient(0deg, rgba(28,31,33,0) 0%, rgba(28,31,33,0.35) 100%);
+    z-index: 3;
```

Comment:
I think for now we can keep it here because the parent of this component is the StepsRouter component. I think we should rename and refactor the StepsRouter component in the future because it's not a router, it just lays down the app structure and it would make more sense to have it separate from the router.